lib/api_remote.ml

```diff
@@ -1,27 +1,26 @@
-open Base
 open Printf
 open Devkit
 open Common
 
 module Github : Api.Github = struct
   let commits_url ~(repo : Github_t.repository) ~sha =
-    String.substr_replace_first ~pattern:"{/sha}" ~with_:("/" ^ sha) repo.commits_url
+    snd @@ ExtLib.String.replace ~sub:"{/sha}" ~by:("/" ^ sha) ~str:repo.commits_url
```

Comment:
Using `snd` can cause some confusion to the reader. Consider:
```
let sub = "{/sha}" in
let by = "/" ^ sha in
let _, output = ExtLib.String.replace ~sub ~by ~str:repo.commits_url in
str
```

Reply:
I feel like OCaml `snd` should be fairly understood? Would it better to put it in the back, like `|> snd` for readability?

Reply:
It is not immediately clear what the first and second element in the tuple represent from the reader's POV. Assigning the pattern-matched value a variable name helps the reader to understand the code better.